settings.py

```diff
@@ -89,11 +89,7 @@
     os.path.join(FILEPATH, 'locale'),
 )
 
-# Do not change, there's a weird bug with Django 1.7 that requires this to be bower_components when using
-# collectstatic
-BOWER_COMPONENTS_ROOT = os.path.join(FILEPATH, 'bower_components')
-
-BOWER_PATH = '/usr/local/bin/bower'
+BOWER_COMPONENTS = os.path.join(FILEPATH, 'bower_components', 'bower_components')
```

Comment:
Will having this in the regular place be easier in the long term? I'm more inclined towards one time switching costs vs. long tail of annoyances. Otherwise, seems fine.